app/src/main/res/values/sdk_version.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="sdk_version">Version 10.10.0</string>
-    <string name="sdk_size">3.20 MB</string>
-    <string name="sdk_release_date">Released 01/10/2020</string>
+    <string name="sdk_version">Version 10.11.0-SNAPSHOT</string>
+    <string name="sdk_size">3.39 MB</string>
```

Comment:
Wow! this a big jump

Reply:
Yeah it has been to those assets for the USP feature on the landing screen!